core/include/engine/Hamiltonian_Heisenberg.hpp

```diff
@@ -33,15 +33,17 @@ class Hamiltonian_Heisenberg : public Hamiltonian
 public:
     Hamiltonian_Heisenberg(
         scalar external_field_magnitude, Vector3 external_field_normal, intfield anisotropy_indices,
-        scalarfield anisotropy_magnitudes, vectorfield anisotropy_normals, pairfield exchange_pairs,
+        scalarfield anisotropy_magnitudes, vectorfield anisotropy_normals, scalarfield cubic_anisotropy_magnitude,
+        pairfield exchange_pairs,
```

Comment:
this doesn't look correctly formatted. Please apply clang-format to the files you changed